content/blog/tinacms-ui-whats-next.md

```diff
@@ -0,0 +1,47 @@
+
+You may have noticed that [tinacms.org](http://tinacms.org) looks different than what's in our videos and what you have running locally. In order to create an amazing inline editing experience for [tinacms.org](http://tinacms.org/) we decided to create an experimental UI. We decided to hide the Sidebar and replace it with a Toolbar at the top of the screen. This new UI was an instant hit with our team! The Toolbar gives us easier access to common actions without having to open and close the Sidebar. We still love the Sidebar, it's easier to setup and is great when you don't want or need inline editing, but it's not always the best experience.
+
+Unfortunately you'll have to wait a little bit to get your hands on the new Toolbar. The creation of the Toolbar was incredibly exciting project for us. We were able to totally re-imagine the user experience of TinaCMS _without having to change any code in the main repository_. This allowed us to quickly experiment with the interface without considering how it could fit into the overall project. Because of this, it only took a few days to get it production ready. This project validated many of code design decision by showing that we could experiment and innovate on top of TinaCMS without having to tear it apart.
+
+Now that Open Authoring with [tinacms.org](http://tinacms.org/) is published. We're getting ready to make it available for everyone. Kendall has already moved over the [new Inline Editing components](https://github.com/tinacms/tinacms/pull/871) and figuring out how the Toolbar will fit in the TinaCMS ecosystem is next. This post lays out the plan for how that will be done.
+
+## `tinacms` as a quick-start package
+
+Currently, the `tinacms` package implements a number of features directly. The Sidebar, Modals, Alerts, and Screen Plugins are all defined directly here. After this change, `tinacms` will simply be the "quick-start" package for developers looking to get started with Tina as fast as possible. The sole purpose of the `tinacms` package will be to provide a pre-configured CMS instance along with a few simple component that adds the CMS, Alerts, Modals, Screens, and either the Sidebar or the Toolbar to the website. This change will give new-comers to Tina the ability to get started quickly, but it will also give other developers the ability to optimize and modify Tina to fit their needs.
+
+A large problem with `tinacms` is the impact it has on the bundle size of the user's website. This will be made worse when we start adding more features to the toolbar. By breaking these pieces up, a `tinacms` user remove this issue by taking advantage of dynamic imports and code splitting in their custom implementation
```

Comment:
"a `tinacms` user remove this issue"? Not sure if you mean to say they *can* remove the issue or something else.